packages/gatsby/content/getting-started/editor-sdks.md

```diff
@@ -80,6 +80,8 @@ With the `.yarn/sdks` in place TypeScript support should work out of the box wit
 
 ##### Supporting go-to-definition et al.
 
+> **Note:** Due to a bug in [Neovim's URI handling](https://github.com/neovim/neovim/pull/14959) go-to-definition is only supported in neovim-nightly or v0.6 (when available).
```

Comment:
This could potentially be updated to v0.5.1 (or something) - trying to confirm if that's an option
https://github.com/neovim/neovim/pull/15017#issuecomment-899474380